components/Container/Container.tsx

```diff
@@ -15,6 +19,9 @@ export type ContainerProps = HTMLAttributes<HTMLElement> & {
   pt?: PaddingType;
   pb?: PaddingType;
   py?: PaddingType;
+  mt?: MarginType;
+  mb?: MarginType;
+  my?: MarginType;
```

Comment:
Curious to when a developer would use this parameter vs classname and what order of significance do they have (which one wins)?

Reply:
I believe whichever classname the props is associated with appears later in the compiled CSS wins 😄 I added these mostly for convenience since it maps nicely to Storyblok datasource. I plan on creating a generic typography component also and will use them as well. It speeds things up for me when there are conditionals, eg, `py={!isMinimal ? 3 : undefined)` (this is for the padding ones)